src/ui/views/notifications.coffee

```diff
@@ -67,7 +67,7 @@ module.exports = (models) ->
         # maybe trigger OS notification
         return if !text or quietIf(c, chat_id)
 
-        if viewstate.showPopUpNotifications
+        if viewstate.showPopUpNotifications and not remote.getCurrentWindow().isVisible()
```

Comment:
This was preventing notifiations to appear in Ubuntu, please check new code and see if worth making this clause windows-specific